src/routes/(console)/account/updateMfa.svelte

```
@@ -81,7 +78,16 @@
     async function updateMfa() {
         try {
             await sdk.forConsole.account.updateMFA({ mfa: !$user.mfa });
-            await invalidate(Dependencies.ACCOUNT);
+
+            if (!$user.mfa && $user.emailVerification && !$factors.email) {
+                try {
+                    await sdk.forConsole.account.createMFAChallenge({
+                        factor: AuthenticationFactor.Email
+                    });
+                } catch (emailError) {}
```

ESLint

[error] Empty block statement.

(no-empty)